libsolidity/analysis/TypeChecker.cpp

```diff
@@ -1760,6 +1760,33 @@ bool TypeChecker::visit(FunctionCall const& _functionCall)
 		}
 	}
 
+	if (functionType->takesSinglePackedBytesParameter())
+	{
+		string generalMessage =
+			"This function only accepts a single 'bytes' argument. Please use the "
+			"abi.* functions to encode the data.";
+
+		if (arguments.size() > 1)
+		{
+			if (v050)
+				m_errorReporter.typeError(_functionCall.location(), generalMessage);
+			else
+				m_errorReporter.warning(_functionCall.location(), generalMessage);
+		}
+		else if (arguments.size() == 1 && !type(*arguments.front())->isImplicitlyConvertibleTo(ArrayType(DataLocation::Memory)))
+		{
+			string msg =
+				generalMessage +
+				" The provided argument of type " +
+				type(*arguments.front())->toString() +
+				" cannot be implictly converted to bytes memory.";
```

Comment:
`implicitly`. Also `"bytes memory"`.

Reply:
Changes this to the more commonly used phrase (in the source): `is not implicitly convertible to  expected type bytes memory`.